docs/patterns/_code-snippets.md

```diff
@@ -0,0 +1,35 @@
+---
+collection: patterns
+title: code snippets
+---
+
+## .code-snippet
+
+Code snippets should be used when presenting the user with a small snippet of code that they will likely want to copy and paste.
+
+```html
+<div class="code-snippet">
+    <input class="code-snippet__input" value="sudo apt-get update" readonly="readonly">
+    <button class="code-snippet__copy-button">Copy to clipboard</button>
+</div>
+```
+
+<div class="code-snippet">
+    <input class="code-snippet__input" value="sudo apt-get update" readonly="readonly">
+    <button class="code-snippet__action">Copy to clipboard</button>
+</div>
+
+## .code-snippet--inverse
+
+
+```html
+<div class="code-snippet code-snippet--inverse">
```

Comment:
👍

On Mon, 15 Aug 2016 at 21:35 Anthony Dillon

<
mailto:Anthony Dillon <notifications@github.com>
> wrote:

a, pre, code, a:link, body { word-wrap: break-word !important; }

In
https://github.com/ubuntudesign/vanilla-framework/pull/460#discussion_r74829935
:

> +<div class="code-snippet"> > + <input class="code-snippet__input" value="sudo apt-get update" readonly="readonly"> > + <button class="code-snippet__copy-button">Copy to clipboard</button> > +</div> > +``` > + > +<div class="code-snippet"> > + <input class="code-snippet__input" value="sudo apt-get update" readonly="readonly"> > + <button class="code-snippet__action">Copy to clipboard</button> > +</div> > + > +## .code-snippet--inverse > + > + > +```html > +<div class="code-snippet code-snippet--inverse">

Question raised
https://github.com/ubuntudesign/vanilla-framework/issues/463
.

—

You are receiving this because you modified the open/close state.

Reply to this email directly,
https://github.com/ubuntudesign/vanilla-framework/pull/460/files/7e42c4ca3cef968e7051929756becbddb9b4f5a2#r74829935
, or
https://github.com/notifications/unsubscribe-auth/AAe24iDMD2F-b8ukFnT224F4ZJJJh5FTks5qgM2CgaJpZM4JjHPR
.